lib/gitsh/tab_completion/dsl/maybe_operation_factory.rb

```diff
@@ -0,0 +1,19 @@
+module Gitsh
+  module TabCompletion
+    module DSL
+      class MaybeOperationFactory
+        attr_reader :child
```

Comment:
Why does this expose `#child`? I don't see it used in any of the implementation.